packages/core/src/Chip/FilterChip/FilterChip.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { space, fontSize } from 'styled-system'
+import { Close } from 'pcln-icons'
+import { ChipContent } from '../ChipContent'
+import { ChipLabel } from '../ChipLabel'
+import { ChipInput } from '../ChipInput'
+
+const getTitle = ({ disabled }) => (disabled ? 'Disabled' : 'Close')
```

Comment:
Should this text be driven by a prop? I could see a need to change this to something like "Remove" in the future, so it'd be nice if "Close" came from a default prop.

Reply:
Updated - added in prop `actionTitle`